### DISCUSSION TITLE
Usuarios con perfil casa no ven algunos datos

### DISCUSSION THREAD

**Discussion by etchegaray** [olvidio/orbix#26] [Ideas]
No salen los datos de número de asistentes ni los ctr organizadores en el "listado de gerentes" de las casas. Adjunto imagen

![image](https://github.com/olvidio/orbix/assets/15702387/e3b11920-3988-47ba-b597-d490f7609538)

**Comment by olvidio** [comment]
Hay que discutir si se activan los dossiers económicos de las casas, y repasar que es lo que deben hacer.
